pom.xml

```diff
@@ -297,6 +313,17 @@
     </resources>
     <pluginManagement>
       <plugins>
+        <plugin>
+          <artifactId>maven-javadoc-plugin</artifactId>
+          <configuration>
+            <skip>true</skip>
+          </configuration>
+        </plugin>
+        <plugin>
+          <groupId>org.codehaus.mojo</groupId>
+          <artifactId>animal-sniffer-maven-plugin</artifactId>
+          <version>1.17</version>
+        </plugin>
```

Comment:
This is to avoid
```
[ERROR] Failed to execute goal org.codehaus.mojo:animal-sniffer-maven-plugin:1.15:check (check) on project nexus-jenkins-plugin: Execution check of goal org.codehaus.mojo:animal-sniffer-maven-plugin:1.15:check failed.: IllegalArgumentException 
```
Fixed by https://github.com/mojohaus/animal-sniffer/commit/db126debbfa70e76a213b7ab63cc418cd1ec1609 in 1.17